infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -163,10 +152,26 @@ cmake_llvm $CMAKE_EXTRA_ARGS \
     -DLLVM_USE_SANITIZER=DataFlow \
     -DCMAKE_INSTALL_PREFIX=/usr/dfsan/
 
-ninja -j $NPROC cxx cxxabi
-ninja install-cxx install-cxxabi
+ninja -j $NPROC install-cxx install-cxxabi
```

Comment:
Why is "ninja -j $NPROC cxx cxxabi" removed ?

Reply:
I'd guess that install does first build the thing and only then installs it, but even if it's so I'd rather prefer to keep explicit build step separate from the install